stylesheets/utils/utilities.css

```diff
@@ -0,0 +1,4 @@
+.flex-center{
+    justify-content: center;
+    align-items:center;
```

Comment:
correct me if  I am wrong but given the name of this class, I assume you'll be using it for `flex` so you can add the `display: flex` property to this util class.

Reply:
Rest everything looks good. Great work 👍

Reply:
Yes, I can also do that. But it can happen that I want an inline-flex at some places so there this utility would not be of any use right

Reply:
Understood the purpose 👍